src/main/java/bio/terra/service/auth/iam/IamProviderInterface.java

```diff
@@ -279,4 +280,18 @@ void overwriteGroupPolicyEmails(
   String signUrlForBlob(
       AuthenticatedUserRequest userReq, String project, String path, Duration duration)
       throws InterruptedException;
+
+  /**
+   * Create a managed resource group in Azure that is associated with a billing profile.
+   *
+   * @param userReq
+   * @param billingProfileId
+   * @param managedResourceGroupCoordinates
+   * @throws InterruptedException
```

Comment:
Could you either provide descriptions for these parameters, or remove them from the docstring if a description is superfluous?